cluster/gce/win1803/BUILD

```diff
@@ -0,0 +1,29 @@
+load("@io_kubernetes_build//defs:build.bzl", "release_filegroup")
+
+# Having the Windows code from the GCE cluster deploy hosted with the release is
+# useful for GKE.  This list should match the list in
+# kubernetes/release/lib/releaselib.sh.
+release_filegroup(
+    name = "gcs-release-artifacts",
+    srcs = [
+        "common.psm1",
+        "configure.ps1",
+        "k8s-node-setup.psm1",
+        "user-profile.psm1",
```

Comment:
I'm updating this directory structure (and adding a file) in https://github.com/kubernetes/kubernetes/pull/74360.